src/library/autodj/dlgautodj.cpp

```diff
@@ -302,6 +307,7 @@ void DlgAutoDJ::autoDJStateChanged(AutoDJProcessor::AutoDJState state) {
         }
         pushButtonFadeNow->setEnabled(false);
         pushButtonSkipNext->setEnabled(false);
+        pushButtonAddRandomTrack->setEnabled(false);
```

Comment:
You can add random tracks independent from the AutoDJ change. Please revert.

Reply:
> This PR does currently not build:
> 
> ```
> /usr/bin/ld: libmixxx-lib.a(dlgautodj.cpp.o): in function `DlgAutoDJ::addRandomTrackButton(bool)':
> dlgautodj.cpp:(.text+0x20): multiple definition of `DlgAutoDJ::addRandomTrackButton(bool)'; libmixxx-lib.a(mocs_compilation.cpp.o):mocs_compilation.cpp:(.text+0x5370): first defined here
> /usr/bin/ld: libmixxx-lib.a(dlgautodj.cpp.o): in function `DlgAutoDJ::addRandomTrackButton(bool)':
> dlgautodj.cpp:(.text+0x28): undefined reference to `AutoDJProcessor::addRandomTrack()'
> /usr/bin/ld: libmixxx-lib.a(mocs_compilation.cpp.o): in function `AutoDJProcessor::qt_static_metacall(QObject*, QMetaObject::Call, int, void**)':
> mocs_compilation.cpp:(.text+0x1e4ce): undefined reference to `AutoDJProcessor::addRandomTrack()'
> /usr/bin/ld: libmixxx-lib.a(autodjprocessor.cpp.o): in function `AutoDJProcessor::controlAddRandomTrack(double)':
> autodjprocessor.cpp:(.text+0xab): undefined reference to `AutoDJProcessor::addRandomTrack()'
> ```

So Sorry about this, It was pretty reckless of me to not notice that, I'll rebase the branch as you suggested.